content/docs/zero/install.mdx

```diff
@@ -101,52 +104,36 @@ docker compose up -d
 Pomerium requires an internet-accessible server and the ability to bind port 443.
 
 </TabItem>
-<TabItem value="manual" label="Manual">
-
-Use the following values as necessary to install or update Pomerium in accordance with your environment:
+<TabItem value="linux" label="Linux">
 
-- **Cluster Token**
-- env: `POMERIUM_ZERO_TOKEN`
-- **Domain Name**
+Install Pomerium with the following shell script:
 
-Pass the provided **Cluster Token** into the `POMERIUM_ZERO_TOKEN` environment variable.
+```bash
+curl https://console.pomerium.app/install.bash | env POMERIUM_ZERO_TOKEN=<CLUSTER_TOKEN> bash -s install
+```
 
-**Domain Name** refers to your cluster's [starter domain](/docs/concepts/clusters#starter-domain).
+Pomerium requires an internet-accessible server and the ability to bind port 443.
 
 </TabItem>
 </Tabs>
 
-## Upgrade Pomerium Zero
+## Update Pomerium Zero
 
-Learn how to upgrade Pomerium Zero.
+Learn how to update Pomerium Zero.
 
 <Tabs>
-<TabItem label="Linux" value="linux">
-
-To update Pomerium in Debian-based Linux systems:
-
-1. Check for new package updates and install Pomerium:
-
-```bash
-$ sudo apt update && sudo apt install pomerium
-```
-
-To update Pomerium in Red Hat-based Linux systems:
-
-1. Check for new package updates
-
-```bash
-$ sudo yum list updates
-```
+<TabItem value="kubernetes-helm" label="Kubernetes (Helm)">
 
-1. Install the latest version of Pomerium:
+To update Pomerium with Helm, run the following command:
 
 ```bash
-$ sudo yum update pomerium
+helm upgrade pomerium-zero oci://docker.io/pomerium/pomerium-zero \
+  --namespace pomerium-zero \
+  --version <X.Y.Z>
```

Comment:
what is `X.Y.Z` in this context? how would user know that? is there a way to just upgrade to latest?

Reply:
This is how the existing docs on this page wrote the version string, I assumed it was something easy to find/replace for releases? not sure